website/templates/public/register.mako

```diff
@@ -43,6 +43,12 @@
         </div>
     %endif
 
+    %if campaign == "agrixiv-preprints":
```

Comment:
This could probably be much more modular. However, we decide just to "patch" this page since it will eventually go away (moved to CAS) 2-3 weeks after Django-OSF goes alive.